monai/networks/nets/magnus.py

```
@@ -0,0 +1,732 @@
+        scale_kernel_sizes: Sequence[int] = (3, 5, 7),
+        norm: Union[str, tuple] = "batch",
+        act: Union[str, tuple] = "relu",
+        dropout: float = 0.0,
+        vit_dropout: float = 0.1,
+        deep_supervision: bool = False,
+        aux_weights: Sequence[float] = (0.4, 0.3, 0.3),
+    ) -> None:
+        super().__init__()
+
+        if spatial_dims not in (2, 3):
+            raise ValueError(f"spatial_dims must be 2 or 3, got {spatial_dims}.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)